process-migration/image.yaml

```diff
@@ -0,0 +1,28 @@
+schema_version: 1
+
+name: "rhpam-7/rhpam75-process-migration"
+description: "Red Hat Process Automation Manager Process Migration 7.5 container image"
+version: "7.5.0"
+from: "ubi8:8-released"
+labels:
+    - name: "com.redhat.component"
+      value: "rhpam-7-rhpam75-process-migration-rhel8-container"
```

Comment:
isn't better just rhpam-process-migration-container?

Reply:
Following the same pattern as the other images. Isn't it correct?

Reply:
For now I say follow the same pattern as the other images We can revisit changing all of them if need be.